packages/inter-protocol/src/priceAggregatorChainlink.js

```diff
@@ -662,19 +699,20 @@ const start = async (zcf, privateArgs) => {
      * directly to manage the price submissions as well as to terminate the
      * relationship.
      *
-     * @param {string} oracleId unique per contract instance
+     * @param {string} oracleAddr Bech32 of oracle operator smart wallet
```

Comment:
The caller of this contract may use an address for the `oracleId`, but the contract doesn't depend on that. See earlier discussion: https://github.com/Agoric/agoric-sdk/pull/6629#discussion_r1054817803